docs/book/src/reference/versions.md

```diff
@@ -63,17 +63,18 @@ These diagrams show the relationships between components in a Cluster API releas
 
 #### Core Provider (`cluster-api-controller`)
 
-|                   | CAPI v0.3 (v1alpha3) | CAPI v0.4 (v1alpha4) | CAPI v1.0 (v1beta1) | CAPI v1.1+v1.2 (v1beta1) |
-|-------------------|----------------------|----------------------|---------------------|--------------------------|
-| Kubernetes v1.16  | ✓                    |                      |                     |                          |
-| Kubernetes v1.17  | ✓                    |                      |                     |                          |
-| Kubernetes v1.18  | ✓                    | ✓ (only workload)    | ✓ (only workload)   | ✓ (only workload)        |
-| Kubernetes v1.19  | ✓                    | ✓                    | ✓                   | ✓ (only workload v1.2)   |
-| Kubernetes v1.20  | ✓                    | ✓                    | ✓                   | ✓                        |
-| Kubernetes v1.21  | ✓                    | ✓                    | ✓                   | ✓                        |
-| Kubernetes v1.22  | ✓ (only workload)    | ✓                    | ✓                   | ✓                        |
-| Kubernetes v1.23* |                      | ✓                    | ✓                   | ✓                        |
-| Kubernetes v1.24  |                      |                      |                     | ✓                        |
+|                   | v0.3 (v1alpha3)      | v0.4 (v1alpha4) 			| v1.0 (v1beta1)      | v1.1 (v1beta1)    | v1.2 (v1beta1)       |
```

Comment:
nit: table misalignment here and below

Reply:
Thx, fixed!

I also changed all columns to use the minimum with (text + 1 space padding on each side)